v22.1/create-security-certificates-custom-ca.md

```diff
@@ -8,13 +8,8 @@ filter_sort: 3
 docs_area: 
 ---
 
-This document discusses the advanced use cases for using Public Key Infrastructure (PKI) security certificates with CockroachDB. PKI certificates are used in CockroachDB for TLS encryption, and node and client authentication.
-
-See also:
-
-- [Public Key Infrastructure (PKI) and Transport Layer Security (TLS)](security-reference/transport-layer-security.html)
-- [Using the CockroachDB CLI to provision a development cluster](manage-certs-cli.html).
-- [Managing CockroachDB security certificates with HashiCorp Vault](manage-certs-vault.html).
+This document discusses the use of advanced [Public Key Infrastructure (PKI)](security-reference/transport-layer-security.html) systems and certificates issued by them with CockroachDB.
+ security certificates with CockroachDB. PKI certificates are used in CockroachDB for TLS encryption and for node and client authentication.
```

Comment:
This is a fragment, perhaps leftover after rewriting.

Reply:
whoops! got it.